Example/TABTestKit_ExampleUITests/ExampleFeature.swift

```diff
@@ -22,7 +22,42 @@ class ExampleFeature: TestBase {
       Then(I: seeTheExampleScreen)
     }
   }
+}
+
+class BiometricTests: XCTestCase {
```

Comment:
If you're creating a new class it should be in a new file

Reply:
This should probably also inherit from `TestBase` like the other class. not suuuper convinced we even need a new class to be honest but if we do we should keep things consistent